i18n/locales/ca-ES/blockly-mooc/tutorialExplorer.json

```diff
@@ -46,7 +47,7 @@
   "filterHeaderTutorialCountPlural": "resultats {tutorial_count}",
   "roboticsButtonText": "View robotics activities",
   "roboticsText": "Got robots? Use these activities and make a tangible Hour of Code for students of any age!",
-  "tutorialSetNoTutorials": "Your search returned no results.\nPlease try changing some filters to see more activities.",
+  "tutorialSetNoTutorials": "Your search returned no results.\nPlease try removing some filters to see more activities.",
```

Comment:
en has "changing" instead of "removing" for some time.. is this change a regression?